rediscluster/utils.py

```diff
@@ -203,3 +203,53 @@ def parse_slots(s):
         nodes.append(node)
 
     return nodes
+
+
+def parse_pubsub_channels(command, res, **options):
```

Comment:
Why have this in utils.py? Why not have this code inside each method in the client class? These methods is not reused elsewhere but in the specific methods.